### PR TITLE
Added JSON request transformer component

### PR DESCRIPTION
**Pull request by damian-arroyo**
Added JSON request transformer to handle json request data in Symfony2 applications.

### CODE REVIEW

File: src/Request/Transformer/JsonRequestTransformerListener.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+class JsonRequestTransformerListener
+{
+    /**
+     * @param GetResponseEvent $event
+     */
+    public function onKernelRequest(GetResponseEvent $event)
+    {
+        $request = $event->getRequest();
+
+        if (!$this->isJsonRequest($request)) {
```
Comment (xurumelous) [inline src/Request/Transformer/JsonRequestTransformerListener.php]: Why not just `if ($request->getContentType() == 'application/json')`?

Comment (damian-arroyo) [inline src/Request/Transformer/JsonRequestTransformerListener.php]: Ok, I'll change that

File: src/Request/Transformer/JsonRequestTransformerListener.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+            if (function_exists('json_last_error_msg')) {
+                $message = json_last_error_msg();
+            }
+
+            throw new JsonRequestTransformerException($message);
+        }
+
+        $request->request->replace($data);
+    }
```
Comment (xurumelous) [inline src/Request/Transformer/JsonRequestTransformerListener.php]: You could use (Linio util's JSON)[https://github.com/LinioIT/util/blob/master/src/Json.php] converter.

**Comment by damian-arroyo** [comment]
Closing since this is old, if needed I can re-write it.

**Pull request closed without merging**